clar2wasm/tests/wasm-generation/arithmetic.rs

```diff
@@ -8,64 +8,69 @@ const TWO_VALUE_OPS: [&str; 2] = ["pow", "mod"];
 const MULTI_VALUE_OPS: [&str; 4] = ["+", "-", "*", "/"];
 
 proptest! {
+  #[ignore]
```

Comment:
There's got to be a better way to ignore all of these that doesn't require adding `#[ignore]` to every test. Can we just exclude this whole directory?

Reply:
Went with Acaccias proposal below to limit the cases rather than ignoring